### PR TITLE
Downstream bug fix

### PR DESCRIPTION
**Pull request by kpratyus**
Making changes so that only one node is added for request header

### CODE REVIEW

File: instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c [hunk 1]
```diff
@@ -721,15 +721,43 @@ static APPD_SDK_STATUS_CODE otel_startInteraction(ngx_http_request_t* r, const c
...
+   part = &r->headers_in.headers.part;
+   header = (ngx_table_elt_t*)part->elts;
+   nelts = part->nelts;
 
    for(int i=0; i<count; i++){
-       h = ngx_list_push(&r->headers_in.headers);
-       if(h == NULL){
-           return;
+
+       char* c = httpHeaders[i];
```
Comment (DebajitDas) [inline instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c:737]: unused variable c?

**Review by DebajitDas — APPROVED** [review]
LGTM

File: instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c (hunk 1 above)
Comment (kpratyus) [reply instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c]: Removed

**Comment by kpratyus** [comment]
> LGTM

We can merge after the jobs are completed

**Pull request merged**